SofaKernel/modules/SofaBaseMechanics/SofaBaseMechanics_test/BarycentricMapping_test.cpp

```diff
@@ -130,25 +129,6 @@ struct BarycentricMapperTriangleSetTopologyTest :  public Test, public Barycentr
 
         EXPECT_EQ(m_convFactor,1./m_gridCellSize);
     }
-
-    void addToHashTable_test()
-    {
-        m_hashTableSize = 1;
```

Comment:
Not possible to adapt the test instead of removing it?

Reply:
@EulalieCoevoet what about the test? Is it really removed or did I missed it or was it not really a test ?

Reply:
It is removed indeed, as I in the end removed the corresponding method.